hack/netpol/README.md

```diff
@@ -0,0 +1,151 @@
+To build this repository, cd to antrea/ and run `go build -o hack/netpol/pkg/main/main.go`.  Then, execute the binary.
```

Comment:
can you make this comment a little more clear? is this for devs? maybe move it to the dev section below?